nemo/core/classes/common.py

```diff
@@ -938,14 +938,19 @@ def _get_hf_hub_pretrained_model_info(cls, model_name: str, refresh_cache: bool
             -   The path to the NeMo model (.nemo file) in some cached directory (managed by HF Hub).
         """
         # Resolve the model name without origin for filename
-        resolved_model_filename = model_name.split("/")[-1] + '.nemo'
+        if model_name.endswith('.nemo'):
```

Comment:
So the repo itself would hand ".Nemo" in the end of the name? Such as Nvidia/abc.nemo?

Reply:
this is for the case when the user specifically specifies .nemo file path in repo (e.g. NLP models have several due to various parallelism configurations)

Reply:
Right. I'm wondering what the signature is like 

ModelClass.from_pretrained("nvidia/???")

What is the value of ??? and how does the user get this value ? Normally it would be something like "Nvidia/stt_en_xyz" where Nvidia is user name / org name and stt_en_xyz is the model card name